docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -207,6 +207,8 @@ Usage: stencil push [<OPTIONS>]
 |`--save [<FILENAME]`           |`-s` | Specify the filename of the saved bundle                                          |
 |`--channel_id [<CHANNEL_ID>]`  |`-c` | Specify the channel ID of the storefront, if the store has multiple storefronts    |
```

Comment:
it is `channel_ids` now. This option allows user to input one or more `ids`. Thus user must specify at least one `id` if they use this flag.